classes/Feature.php

```diff
@@ -251,15 +251,15 @@ public static function nbFeatures($idLang)
      *
      * @return int Feature ID
      */
-    public static function addFeatureImport($name, $position = false)
+    public static function getFeatureImport($name, $position = false, $add_if_not_exists = true)
```

Comment:
It is a public method, therefore it is covered by our backward policy promise, which means if you want to rename a method, then you can create a new one and deprecate the old one. That is allowing us to not break every function and have a stable api for module developers.

Please note that this is not applicable if your method is private.


Adding a optional parameter with a default value is fine, as the code will still be working as it used to.

Reply:
Good point, I will create a new one and deprecate the original. The label addFeatureImport has no sense if we don't actually add something.